src/PointwiseFunctions/ScalarTensor/ScalarCharge.hpp

```diff
@@ -0,0 +1,77 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "ApparentHorizons/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace ScalarTensor {
+
+/// @{
+/*!
+ * \brief The scalar charge per unit solid angle.
+ *
+ * \details This function calculates the integrand of:
+ * \f{align*}
+ * q = - \dfrac{1}{4 \pi} \oint dA \Phi_i n^{i},
+ * \f}
+ * where \f$ n^{i} \f$ is the unit (outward) normal of the surface.
+ *
+ * For a spherically symmetric scalar, this value will coincide with the value
+ * as extracted from the $r^{-1}$ decay in
+ * \f[ \Psi \sim \phi_\infty + q / r + \cdots~. \f]
+ */
+template <typename Frame>
```

Comment:
Will you need instantiations for the different frames? Else you can just remove the `Frame` parameter altogether. `tnsr::i` defaults to inertial frame.

Reply:
Also removed the `Frame` parameter